Source/WebGPU/WGSL/AST/ASTStructure.h

```diff
@@ -66,37 +41,37 @@ enum class StructRole : uint8_t {
     ArgumentBuffer,
 };
 
-class StructDecl final : public Decl {
+class Structure final : public Declaration {
```

Comment:
Just a nit, but I think this is one of the cases according to the style guide where the abbreviation is the preferred form. Maybe @litherum has strong feelings about this?